app/models/delivery.js

```diff
@@ -10,7 +10,7 @@ export default DS.Model.extend({
   fromUser: DS.belongsTo('DukeDsUser'),
   toUser: DS.belongsTo('DukeDsUser'),
   state: DS.attr('string'),
-  transfer: DS.belongsTo('DukeDsProjectTransfer'),
+  transferId: DS.attr('string'),
```

Comment:
Are they no longer related?

Reply:
I was concerned about a circular dependency (Transfer -> Delivery -> Transfer).
I believe it is lazily loaded so that might not be a problem.